File: Examples/CSharp/Loading-and-Saving/CreateEmpty3DDocument.cs
using System;
using System.IO;
using System.Collections;
using Aspose.ThreeD;
namespace Aspose._3D.Examples.CSharp.Loading_Saving
{
    class CreateEmpty3DDocument
    {
        public static void Run()
        {
            // ExStart:CreateEmpty3DDocument
            // The path to the documents directory.
            var output = RunExamples.GetOutputFilePath("document.fbx");

            // Create an object of the Scene class
            Scene scene = new Scene();
            // Save 3D scene document
            scene.Save(output, FileFormat.FBX7500ASCII);
            // ExEnd:CreateEmpty3DDocument

            Console.WriteLine("\nAn empty 3D document created successfully.\nFile saved at " + output);
        }
    }
}

File: Examples/VisualBasic/Loading-and-Saving/CreateEmpty3DDocument.vb
Imports System.IO
Imports System.Collections
Imports Aspose.ThreeD
Namespace Loading_Saving
    Public Class CreateEmpty3DDocument
        Public Shared Sub Run()
            ' ExStart:CreateEmpty3DDocument
            ' The path to the documents directory.
            Dim MyDir As String = RunExamples.GetDataDir()
            MyDir = MyDir & Convert.ToString("document.fbx")

            ' Create an object of the Scene class
            Dim scene As New Scene()
            ' Save 3D scene document
            scene.Save(MyDir, FileFormat.FBX7400ASCII)
            ' ExEnd:CreateEmpty3DDocument

            Console.WriteLine(Convert.ToString(vbLf & "An empty 3D document created successfully." & vbLf & "File saved at ") & MyDir)
        End Sub
    End Class
End Namespace